demo/app.component.ts

```diff
@@ -675,6 +690,11 @@ export class AppComponent implements OnInit {
   gaugeShowAxis: boolean = true;
   gaugeValue: number = 50; // linear gauge value
   gaugePreviousValue: number = 70;
+  gaugeMargin: boolean = false;
```

Comment:
Let's rename these to be margin, marginLeft, marginRight, marginTop, and marginBottom. We may want to expose the margins on other charts in the future, so they should be generic.